org.eclipse.jdt.core/model/org/eclipse/jdt/internal/core/builder/CompilationParticipantResult.java

```diff
@@ -143,9 +164,9 @@ public void recordNewProblems(CategorizedProblem[] newProblems) {
 	this.problems = merged;
 }
 
-void reset(boolean detectedAnnotations) {
+void reset(AnnotationBinding[] annotations) {
```

Comment:
This introduces a new warning about the parameter hiding a field.

Reply:
This should be fixed now!